src/components/Computer/index.js

```diff
@@ -1,8 +1,49 @@
 import React from 'react';
 import './computer.css'
+import paper from './assets/paper.png';
+import rock from './assets/rock.png';
+import scissors from './assets/scissors.png';
+import defaultImg from './assets/defaultImg.jpg';
+
+import {getRandomChoice} from '../../utils';
+
 const Computer = () =>{
+const choice = getRandomChoice();
+
```

Comment:
This function should be called from the parent componenet which would decide the winner, otherwise how would you pass the choice from the bot/computer, a function could be passed a `prop` to the child which could be called with the result of choice. 
More info on that:
[React Doc](https://reactjs.org/docs/lifting-state-up.html)
[StackOverflow question](https://stackoverflow.com/questions/38394015/how-to-pass-data-from-child-component-to-its-parent-in-reactjs )